src/frontend/components/ErrorComponents/index.tsx

```diff
@@ -34,110 +36,134 @@ const ErrorContent = styled.div`
-      id: 'components.ErrorComponent.upload.text',
+      id: 'components.FullScreenError.upload.text',
     },
     title: {
       defaultMessage: 'Failed to upload your video file',
       description: 'Title for the video upload error page',
-      id: 'components.ErrorComponent.upload.title',
+      id: 'components.FullScreenError.upload.title',
     },
   }),
   liveIncompatible: defineMessages({
     text: {
       defaultMessage:
         'You can try again later. You may want to check your Internet connection quality.',
       description: 'Helpful text for the Upload error page',
-      id: 'components.ErrorComponent.upload.text',
+      id: 'components.FullScreenError.upload.text',
     },
     title: {
       defaultMessage: 'Live mode incompatible with this object',
       description: 'Title when the object is incompatible with live mode',
-      id: 'components.ErrorComponent.liveIncompatible.title',
+      id: 'components.FullScreenError.liveIncompatible.title',
     },
   }),
   liveInit: {
     text: {
       defaultMessage:
         'We could not make sure you are allowed to modify a live. Please check your settings and/or try again.',
       description: 'Title for the live permission error page',
-      id: 'components.ErrorComponent.liveInit.text',
+      id: 'components.FullScreenError.liveInit.text',
     },
     title: {
       defaultMessage: 'Failed to authenticate your permission to modify a live',
       description: 'Helpful text for the upload permission error page',
-      id: 'components.ErrorComponent.liveInit.title',
+      id: 'components.FullScreenError.liveInit.title',
     },
   },
   liveToVod: {
     text: {
       defaultMessage: 'Failed to publish a VOD',
       description: 'Error title when publish a live to VOD fails.',
-      id: 'components.ErrorComponent.liveToVod.title',
+      id: 'components.FullScreenError.liveToVod.title',
     },
     title: {
       defaultMessage:
         'We could not publish this video as a VOD. Please retry later',
       description: 'Error message when publish a live to VOD fails.',
-      id: 'components.ErrorComponent.liveToVod.text',
+      id: 'components.FullScreenError.liveToVod.text',
     },
   },
 };
 
-export const ErrorComponent = (props: ErrorComponentProps) => {
-  return (
-    <ErrorComponentStyled>
-      <ErrorContent>
-        <H2>
-          <FormattedMessage {...messages[props.code].title} />
-        </H2>
-        <p>
-          <FormattedMessage {...messages[props.code].text} />
-        </p>
-      </ErrorContent>
-    </ErrorComponentStyled>
-  );
-};
+export const ErrorMessage = ({ code }: ErrorComponentsProps) => (
+  <Box direction="column">
+    <Heading level={6}>
+      <FormattedMessage {...messages[code].title} />
+    </Heading>
+    <Paragraph>
+      <FormattedMessage {...messages[code].text} />
+    </Paragraph>
+  </Box>
+);
+
+export const FullScreenError = ({ code }: ErrorComponentsProps) => (
```

Comment:
You are sometimes using the type `React.FC`, sometimes not. We should decide what to adopt. Since now we didn't use it. Should we ?
This comment is not directly related to his PR but we have to discuss about this.

Reply:
IMO it’s better than just defining the type for the props.

- it checks the component as a whole (for example forcing you to return something valid)
- it’s good context in the component file 
- it reminds you to import react which is implicitly required


Since it’s free, I see little reason not to use it. I should do it in this PR but I’m not sure it’s worth it to specifically add it back to the whole app right this moment.
Wdyt ?

Reply:
Maybe @jbpenrath has an opinion also 😅

Reply:
I approve the PR but we can continue the discussion here.

Reply:
From a typing point of view, I think it's better to type as strong as possible and `React.FC` fullfill this point.

But in other hand, about readability, I find a component using `React.FC` a little bit harder to read and I think it is enough to type correctly props attemped by the functional component.

So to resume, IMO, from strict code quality point of view it is better to use `React.FC`, but from practice and day to day coding, do not use it is not risky, more readable and easier (because we do not have to remind ourself to include something ^^).

The situation is the same in Richie project so I will also update Richie accordingly to the conclusion of this dicussion.

Reply:
As usual Kent C. Dodds comes through with a good argument: https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript

Using `React.FC` makes the component accept a `children` props, which is wrong for most of our components that do not accept it.
For those that need children, we can use `React.PropsWithChildren`.